packages/eas-cli/src/build/metadata.ts

```diff
@@ -36,6 +36,17 @@ export interface IosMetadataContext {
   buildConfiguration?: string;
 }
 
+// TODO(JJ): Replace this with the getRuntimeVersionNullable function in @expo/config-plugins
+function getRuntimeVersionNullable(
+  ...[config, platform]: Parameters<typeof Updates.getRuntimeVersion>
+): string | null {
+  try {
+    return Updates.getRuntimeVersion(config, platform);
+  } catch (e) {
+    return null;
```

Comment:
```suggestion
    Log.debug(err);
    return null;
```
So we can check the error in debug mode.